.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,33 @@
+#### What does this PR do?
```

Comment:
Since this is a CX template we can use CX groups and not Codice ones but I am okay with this too.